.github/workflows/test.yml

```diff
@@ -85,12 +86,31 @@ jobs:
       - id: files
         name: Run the action
         uses: ./
+        with:
+          include: '.*'
 
-      - name: Print steps context output
+      - name: Print files
```

Comment:
[nitpick] I think the name was accurate, but it is not very important. Does someone else has an opinion about this?

Reply:
I wanted to have a different name since I had duplicated the step with the exclude filter.